lib/lhm/chunker.rb

```diff
@@ -52,6 +52,9 @@ def execute
         break if @start == @limit
       end
       @printer.end
+    rescue => e
+      @printer.exception(e) if @printer.respond_to?(:exception)
```

Comment:
Using `respond_to?` because people may be using their own printers that do not define this method